pkg/guardian/redis_conf_store.go

```diff
@@ -11,18 +12,25 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/go-redis/redis"
 	"github.com/pkg/errors"
+	"github.com/bsm/redislock"
```

Comment:
Is this worth the additional dependency? It seems like a very simple implementation.

https://github.com/bsm/redislock/blob/master/redislock.go

Reply:
It's running [Lua scripts](https://github.com/bsm/redislock/blob/master/redislock.go#L18).